lib/cli/cardano-wallet-cli.cabal

```diff
@@ -41,11 +41,13 @@ library
     , fmt
     , http-client
     , iohk-monitoring
+    , iso8601-time
```

Comment:
I don't think this is a necessary dependency? 
What about:

https://hackage.haskell.org/package/time-1.9.2/docs/Data-Time-Format-ISO8601.html 

 ?

Reply:
> I don't think this is a necessary dependency?
> What about:
> 
> https://hackage.haskell.org/package/time-1.9.2/docs/Data-Time-Format-ISO8601.html

I also saw that. However, it depends on `>= time-1.9`, which currently isn't available on any version of Stackage. (The latest available is `1.8.0.2`.)

I'll experiment with using an `extra-dep` and report back.

Reply:
So it seems that many of the libraries on which we depend all require `time < 1.9`, even in the latest version of Stackage.

Reply:
It would be really nice to use this module (`Data.Time.Format.ISO8601`) from `time >= 1.9`, but I'm not sure it's worth the extra hassle until `time >= 1.9` is included in a Stackage LTS release.

I did (experimentally) try to get things to compile with `time-1.9.3` and succeeded in producing a build, but I had to resort to using `allow-newer: true` within `stack.yaml` to make this happen, which I'm sure we'd prefer to avoid.